src/dsp_tools/commands/excel2json/properties.py

```diff
@@ -165,6 +152,33 @@ def _do_property_excel_compliance(df: pd.DataFrame, excelfile: str) -> None:
         raise InputError("\n\n".join(msg))
 
 
+def _add_optional_columns(df: pd.DataFrame) -> pd.DataFrame:
+    optional_col_set = {
+        "label_en",
+        "label_de",
+        "label_fr",
+        "label_it",
+        "label_rm",
+        "comment_en",
+        "comment_de",
+        "comment_fr",
+        "comment_it",
+        "comment_rm",
+        "subject",
+    }
+    in_df_cols = set(df.columns)
+    if "subject" not in in_df_cols:
+        warnings.warn(
+            "It is possible to add a constraint to the subject of a property.\n"
+            'To do this you can enter a resource name in the column "subject".'
```

Comment:
This message is not ideal:
- It's a warning, but then the user is only told that it is possible to do something. For me, a warning only makes sense if we tell the user that he SHOULD do something, and that there are negative consequences if he doesn't. So we should also discuss in the RDU team if the subject constraint is something completely optional, or rather something that should be done if ever possible.
- If I understand correctly, the code at this place only checks if the column is present or not. So it doesn't make sense to say that the user `can enter a resource name in the column "subject"`, if the column is not there.